Packs/IntegrationsAndIncidentsHealthCheck/Scripts/CopyLinkedAnalystNotes/CopyLinkedAnalystNotes.py

```diff
@@ -55,6 +55,9 @@ def main():
 
     if linked_incident:
         main_integration_grid, main_incident_grid = create_grids(custom_fields, linked_incident)
+        if len(main_incident_grid) != 1:
```

Comment:
Why is this relevant?
Sorting 1 column is still sorting.

Reply:
I saw that when main_incident_grid is empty (i.e there are no rows) the list is not actually empty. It contains one element that has a dictionary like this - {"commandname":0} . So there is no creationdate in this case (sort will fail).
This condition helps me "pass" this case.

Reply:
Please try to understand where this object is coming from and why is it there to begin with.

Reply:
<img width="1709" alt="Screen Shot 2020-10-18 at 12 09 05" src="https://user-images.githubusercontent.com/71635916/96363088-ba853700-113a-11eb-979c-3a325da79de8.png">

The grid looks like this when it is empty. Should I try to change it?